packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -1,631 +1,249 @@
+      this._replaceMenuIcon();
     });
-
-    /**
-     * Search form integration.
-     */
-    this._setupSearchForm();
-
-    /**
-     * Decide `<vaadin-tabs>` initial orientation.
-     */
-    this._updatedWide();
-
-    /**
-     * Highlight current menu item.
-     */
-    this._highlightCurrentMenuItem();
-
-    /**
-     * Handle `<vaadin-context-menu-overlay>`.
-     * Check every 1000ms the scroll position.
-     */
-    window.addEventListener(
-      'scroll',
-      throttle(() => {
-        this._maybeCloseContextMenuOverlay();
-      }, 1000)
-    );
-
-    document.body.addEventListener('vaadin-overlay-open', this._boundOnOverlayOpen);
-
-    super.firstUpdated(changedProperties);
   }
 
-  /**
-   * Adjusts the target element's position to fit within the available width.
-   *
-   * @param {HTMLElement} target - The target element to adjust.
-   * @return {string|boolean} The adjusted position value in pixels or `false` if no adjustment is needed.
-   */
-  static _adjustToWidth(target) {
-    if (target.clientWidth + target.offsetLeft > visualViewport.width) {
-      return `${
-        target.offsetLeft - (target.clientWidth + target.offsetLeft - visualViewport.width)
-      }px`;
+  updated(changes) {
+    if (changes.has('groups') || changes.has('wide')) {
+      this._updateContextMenuItems();
+      this._replaceMenuIcon();
     }
-    return false;
-  }
-
-  /*
-   * Setup listeners on nav items to handle overlays events.
-   */
-  _setupSlottedMenuItems() {
-    const navItems = this.querySelectorAll('vaadin-tab');
-    const overlaysWrapper = this.overlaysWrapperElement;
-
-    navItems.forEach((tab) =>
-      tab.addEventListener('click', (e) => {
-        const overlays = overlaysWrapper.querySelectorAll(
-          'vaadin-context-menu-overlay[theme="cxl-marketing-nav"]'
-        );
-        const parentOffset = tab.parentElement.getBoundingClientRect();
-        overlaysWrapper.style.top = `${parentOffset.y + 0.75 * tab.clientHeight}px`;
-        overlaysWrapper.style.left = `${e.clientX}px`;
-
-        [...overlays].forEach((overlay) => {
-          overlay.close();
-        });
-      })
-    );
   }
 
-  _onOverlayClose(e) {
-    const overlay = e.target;
-    if (overlay.parentElement !== this.overlaysWrapperElement) return;
+  render() {
+    const groups = this.wide ? this.groups : this.mobileGroups;
 
-    document.body.appendChild(overlay);
-    if (!this.overlaysWrapperElement.children.length) {
-      this.overlaysWrapperElement.toggleAttribute('hidden', true);
-    }
+    return html`
+      ${groups.map((group) => {
+        const { name, items } = group;
+        return html`
+          <nav id="menu-${name}-items" ?minimal=${!this.wide}>
+            <div class="container">
+              ${name === 'global' || !this.wide
+                ? html`
+                    <vaadin-menu-bar-button theme="tertiary cxl-marketing-nav">
+                      <a href="https://cxl.com">
```

Comment:
Logo URL can't be hardcoded. We can have fallback to https://cxl.com but it should be passed by argument to the component. We have WP instances and we can't redirect people from https://cxl.com/institute to https://cxl.com